src/uwtools/rocoto.py

```diff
@@ -348,9 +348,7 @@ def _config_validate(self, config: Union[dict, YAMLConfig, Optional[Path]]) -> N
         :raises: UWConfigError if config fails validation.
         """
         schema_file = resource_path("jsonschema/rocoto.jsonschema")
-        ok = validate_yaml(schema_file=schema_file, config=config)
-        if not ok:
-            raise UWConfigError("YAML validation errors")
+        validate_yaml(schema_file=schema_file, config=config)
```

Comment:
This is a great simplification! I was excited to see how this method would change with the update and it looks much cleaner now that `validate_external` which is imported as `validate_yaml` here raises an error as it's return value instead a bool now.